sync/field.field.node.event.field_processed_json.yml

```diff
@@ -0,0 +1,19 @@
+uuid: 57435f2b-6e19-43ca-9ccf-7cc422502720
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_processed_json
+    - node.type.event
+id: node.event.field_processed_json
+field_name: field_processed_json
+entity_type: node
+bundle: event
+label: 'Processed (JSON)'
+description: ''
+required: false
+translatable: true
+default_value: {  }
+default_value_callback: ''
+settings: {  }
+field_type: string_long
```

Comment:
I'm not sure how does this get mapped at the MySQL level, but should it be a JSON column (we are on 5.7+)?

Reply:
This is a Drupal field so we can only use the available field types. If we were using a custom schema we might be able to use JSON column type.